GVFS/GVFS.FunctionalTests/Tests/GitCommands/StatusTests.cs

```diff
@@ -36,6 +36,23 @@ public void DeleteThenCreateThenDeleteFile()
             this.ValidateGitCommand("status");
             this.DeleteFile(srcPath);
             this.ValidateGitCommand("status");
+        }
+
+        [TestCase]
+        public void CreateFileWithoutClose()
+        {
+            string srcPath = @"CreateFileWithoutClose.md";
+            this.CreateFileWithoutClose(srcPath);
+            this.ValidateGitCommand("status");
+        }
+
+        [Ignore("This test exposes a bug we don't yet have a fix for")]
```

Comment:
The fix here will be addressed in a separate.  I wanted to validate the issue / my understanding which is why I'm adding the functional test now.

Reply:
You could include in the ignore message that https://github.com/Microsoft/VFSForGit/issues/209 should fix this issue.